drivers/timer/nrf_rtc_timer.c

```diff
@@ -40,6 +40,13 @@ BUILD_ASSERT(CHAN_COUNT <= RTC_CH_COUNT, "Not enough compare channels");
 #define ANCHOR_RANGE_END (7 * COUNTER_SPAN / 8)
 #define TARGET_TIME_INVALID (UINT64_MAX)
 
+#define NRF_RTC_INT_ALL_MASK (RTC_INTENSET_TICK_Msk |		\
```

Comment:
The `NRF_RTC_` prefix should be reserved for stuff defined in `<hal/nrf_rtc.h>`. Please use some other name.

Reply:
Thanks, sorry for the mess. Indeed there was no need of a new define and I used now only the defines in the interface nrf_rtc.h.